docs/getting-started.md

```diff
@@ -0,0 +1,629 @@
+The Custom Embeds workflow is made up of "panels." Currently, these panels must reside on the *public internet*. This means that they are accessible by anyone in the world with internet access. Therefore we ask that you adhere to a few key restrictions:
+
+* The content exposed in these panels should only include published, public content. You should not search against unpublished content, data that is private to an organization or an individual, or content with legal restrictions placed on it.
+* None of these panels should write or modify the data they expose in the data's original source. The panels function only by sending and receiving data from Composer via the user's browser. All edited data should be saved within the Composer ANS document, and nowhere else.
+
+## Steps
+
+### 1. Identify or create endpoints for your external data.
+
+Your external data source should provide public GET endpoints as specified above for fetching and searching for content.
+These may already exist, or in some cases you may be able to create them yourself.
+
+This guide will use the [OMDB API](https://www.omdbapi.com/) as an example data source.
+
+Here's an example endpoint for fetching content by ID:
+
+* https://www.omdbapi.com/?apikey=YOUR_API_KEY_HERE&plot=full&i=tt3969158
+
+Here's an example endpoint for search for content by name:
+
+* https://www.omdbapi.com/?apikey=YOUR_API_KEY_HERE&s=Jurassic
+
+> **What about *my* content?**
+>
+> To implement an Arc Composer Custom Embed for your external content, your data source will need to:
+>
+> * ...expose an HTTP GET endpoint for fetching content metadata by id
+> * ...expose an HTTP GET endpoint for querying for content in your content source as a Composer user would
+>   * ...**without** exposing confidential, unpublished or otherwise private content or data
+>   * ...and **without** submitting new contnet or modifying content in the external data source
+>
+> These last two restrictions are in place because your endpoints will be indirectly exposed on the **public internet**,
+> so it important that they do not expose anything that is not already discoverable on the web.
+
+
+
+### 2. Build a content source in PageBuilder Fusion
+
+We'll need two content sources set up in Fusion.
+
+The first content source will be used for rendering a single embed on the website as well as the view and edit panels. It will need to fetch a piece of content by id and return its metadata.
+
+Let's modify the existing movie-find content source from the Fusion recipes to accept an `id` parameter in addition to the `title` parameter.
+
+```javascript
+/*    /content/sources/movie-find.js    */
+
+import { OMDB_API_KEY } from 'fusion:environment'
+
+const resolve = (query) => {
+  const requestUri = `https://www.omdbapi.com/?apikey=${OMDB_API_KEY}&plot=full`
+
+  if (query.hasOwnProperty('movieTitle')) return `${requestUri}&t=${query.movieTitle}`
+  if (query.hasOwnProperty('imdbID')) return `${requestUri}&i=${query.imdbID}`
```

Comment:
Do you want this to be an `else if`? Not sure if you want to send both params if they are both specified, or just one or the other.